it/topics/README.md

```diff
@@ -1,5 +1,22 @@
-# Il soggetto
+# 💡 Argomenti
+> Guide specifiche per i linguaggi
 
-[:gb: Topics](/en/topics/README.md)
+Consulta le cartelle in questo repository per conoscere una linguaggio specifico.
 
-Help wanted! Add translations here.
+Alcuni punti salienti:
+
+- Scripting con linguaggi vecchi
+	- [JavaScript](scripting_languages/JavaScript)
```

Comment:
These links point to folders which don't exist yet in :it: . Do you have plan for that, otherwise can we link to :gb: versions in the meantime?

Reply:
@MichaelCurrin as I'm concerned for now it would be better to put the links in the English version, so that they work, and then modify them in the future to adapt to the Italian version.